includes/ThanksPermissions.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace MediaWiki\Extension\Thanks;
+
+use MediaWiki\MediaWikiServices;
+use MobileContext;
+
+class ThanksPermissions {
+
+	private static function isMobile() {
+		global $wgFandomMobileSkinEnabled;
+
+		if ( $wgFandomMobileSkinEnabled && class_exists( 'MobileContext' ) ) {
```

Comment:
I get that this is just a copy of `isMobile` from UCP, but in Thanks context, we actually don't need this check. It's always set to `true` unless the page is one of the following: https://github.com/Wikia/unified-platform/blob/19334d2874ed2572382cef01858dc73f652eeff5/config/dynamic-expansions.php#L572-L576